chat.html

```diff
@@ -408,7 +415,7 @@
 
             let el_badges = createElement('span', {'class': 'msg-badges'});
             let el_pronoun = createElement('span', {'class': 'msg-pronoun' }, pronoun);
-            let el_message = createElement('span', {'class': 'msg-text'});
+            let el_message = createElement('span', {'class': 'msg-text', 'style': `color: ${message_color}`});
```

Comment:
This would produce `style="color: null"` if the color isn't set. Ideally this would have a conditional and not set anything.